scene/resources/particle_process_material.cpp

```diff
@@ -1567,16 +1567,25 @@ void ParticleProcessMaterial::set_emission_shape(EmissionShape p_shape) {
 	emission_shape = p_shape;
 	notify_property_list_changed();
 	_queue_shader_change();
+#ifdef TOOLS_ENABLED
+	emit_changed();
+#endif
```

Comment:
In that case cc @clayjohn @QbieShay

Reply:
Indeed, there's a conflict here because it was agreed in a meeting to remove the separate signal and use `changed` instead.

Reply:
Okay I changed it back to the `emission_shape_changed` signal with still editor flags around it. I don't see a reason to have it for runtime as well since the visualization is editor-only